packages/alert-dialog/package.json

```diff
@@ -0,0 +1,77 @@
+{
+    "name": "@spectrum-web-components/alert-dialog",
+    "version": "0.0.1",
+    "publishConfig": {
+        "access": "public"
+    },
+    "description": "Web component implementation of a Spectrum design AlertDialog",
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/adobe/spectrum-web-components.git",
+        "directory": "packages/alert-dialog"
+    },
+    "author": "",
+    "homepage": "https://adobe.github.io/spectrum-web-components/components/alert-dialog",
+    "bugs": {
+        "url": "https://github.com/adobe/spectrum-web-components/issues"
+    },
+    "main": "./src/index.js",
+    "module": "./src/index.js",
+    "type": "module",
+    "exports": {
+        ".": {
+            "development": "./src/index.dev.js",
+            "default": "./src/index.js"
+        },
+        "./package.json": "./package.json",
+        "./src/AlertDialog.js": {
+            "development": "./src/AlertDialog.dev.js",
+            "default": "./src/AlertDialog.js"
+        },
+        "./src/alert-dialog.css.js": "./src/alert-dialog.css.js",
+        "./src/index.js": {
+            "development": "./src/index.dev.js",
+            "default": "./src/index.js"
+        },
+        "./sp-alert-dialog.js": {
+            "development": "./sp-alert-dialog.dev.js",
+            "default": "./sp-alert-dialog.js"
+        }
+    },
+    "scripts": {
+        "test": "echo \"Error: run tests from mono-repo root.\" && exit 1"
+    },
+    "files": [
+        "**/*.d.ts",
+        "**/*.js",
+        "**/*.js.map",
+        "custom-elements.json",
+        "!stories/",
+        "!test/"
+    ],
+    "keywords": [
+        "spectrum css",
+        "web components",
+        "lit-element",
+        "lit-html"
+    ],
+    "dependencies": {
+        "@spectrum-web-components/base": "^0.37.0",
+        "@spectrum-web-components/button": "^0.37.0",
+        "@spectrum-web-components/button-group": "^0.37.0",
+        "@spectrum-web-components/divider": "^0.37.0",
+        "@spectrum-web-components/icons-workflow": "^0.37.0",
+        "@spectrum-web-components/shared": "^0.37.0"
+    },
+    "devDependencies": {
+        "@spectrum-css/alertdialog": "1.0.4"
```

Comment:
There is one newer if we want before merging this.

Reply:
Updated